File: java/get-localized-content/getting_localized_content_language.java
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("KONTENT_AI_ENVIRONMENT_ID");

// Tip: Create strongly typed models according to https://kontent.ai/learn/net-strong-types
// Registers the model class for articles
client.registerType(Article.class);

// Gets a specific article in Spanish
CompletionStage<Article> item = client.getItem(
	"about_us",
	Article.class,
	DeliveryParameterBuilder.params()
		.language("es-ES")
		.build()
	
);

File: ts/get-localized-content/getting_localized_content_language.ts
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { createDeliveryClient } from '@kontent-ai/delivery-sdk';
import { Article } from './models/Article';

const deliveryClient = createDeliveryClient({
  environmentId: '8d20758c-d74c-4f59-ae04-ee928c0816b7',
});

// Gets a specific article in Spanish
const response = await deliveryClient.item<Article>('about_us')
  .languageParameter('es-ES')
  .toPromise();